README.md

```diff
@@ -52,7 +52,15 @@ Connect an Android device to your development machine or use android emulator pr
 
 | Light mode | Dark mode |
 |--|--|
-| ![alt text](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/android_simform_sample_app/master/screenshots/outputLightMode.png) | ![alt text](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/android_simform_sample_app/master/screenshots/outputDarkMode.png) |
+| ![alt text](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/android_simform_sample_app/master/readmeresources/outputLightMode.png) | ![alt text](https://raw.githubusercontent.com/SimformSolutionsPvtLtd/android_simform_sample_app/master/readmeresources/outputDarkMode.png) |
+
+### Pagination
```

Comment:
Pagination is part of tech stack. Can you please move it into "Tech stack" section of this README.